README.md

```diff
@@ -117,7 +114,6 @@ jobs:
       - uses: styfle/cancel-workflow-action@0.9.1
         with:
           all_but_latest: true
-          access_token: ${{ github.token }}
```

Comment:
Does this work for private repos?

I think the user needs to generate a token and supply a secret here so its best to leave this as-is

Reply:
People copy/paste things like this from the README.
Didn't test this, but if it would not work, this change would still benefit most of users